setup.py

```diff
@@ -1,6 +1,33 @@
 #!/usr/bin/env python
 """
-  TODO: this
+<Program Name>
+  setup.py
+
+<Author>
+  Santiago Torres <santiago@nyu.edu>
+  Lukas Puehringer <lukas.puehringer@nyu.edu>
+
+<Started>
+  May 23, 2016
+
+<Copyright>
+  See LICENSE for licensing information.
+
+<Purpose>
+  setup.py script to install in-toto framework and in-toto scripts
+
+  # System Dependencies
```

Comment:
Our *system* doesn't depend on git, or pip per se. I think this is from the readme, but we could probably call these "development requirements" rather than "system dependencies"? What do you think?

Reply:
Makes sense to not list those under "system dependencies". What about the oxymoronic "optional requirements"? Because at least pip is not really a "development requirement" either.

Reply:
I think probably going with debian's `recommended tools` for pip/git, and the others as actual `requirements`. What do you think?

Reply:
Sounds good to me, I'd vote for adding `virtualenvs` to recommended tools as well

Reply:
sounds good 👍